docs/data/charts/hooks/UseBarSeries.tsx

```diff
@@ -0,0 +1,97 @@
+import * as React from 'react';
+import { useBarSeries } from '@mui/x-charts/hooks';
+import { BarChart } from '@mui/x-charts/BarChart';
+import { useTheme } from '@mui/material/styles';
+
+const dataset = [
+  { month: 'Jan', revenue: 4000, expenses: 2400 },
+  { month: 'Feb', revenue: 3000, expenses: 1398 },
+  { month: 'Mar', revenue: 5000, expenses: 2800 },
+  { month: 'Apr', revenue: 7000, expenses: 3908 },
+  { month: 'May', revenue: 6000, expenses: 4800 },
+  { month: 'Jun', revenue: 3000, expenses: 8000 },
+];
+
+const inUSD = (value: number | null | undefined) => {
+  if (value == null) {
+    return '';
+  }
+  return `$${value.toLocaleString()}`;
+};
+
+function BarSeriesInfo() {
+  const barSeries = useBarSeries();
+  const theme = useTheme();
+
+  const profit = dataset.reduce(
+    (acc, curr) => acc + (curr.revenue - curr.expenses),
+    0,
+  );
+
+  return (
+    <div
+      style={{
+        position: 'absolute',
+        top: 20,
+        left: 70,
+        background: `color(from ${theme.palette.text.primary} srgb r g b / ${theme.palette.mode === 'dark' ? 0.2 : 0.1})`,
+        padding: 12,
+        borderRadius: 4,
+        fontSize: '12px',
+        border: '1px solid #e0e0e0',
+        zIndex: 10,
+      }}
+    >
+      <h4 style={{ margin: '0 0 8px 0', fontSize: '14px' }}>Half Year Details</h4>
+      {Object.entries(barSeries).map(([seriesId, series]) => (
+        <div key={seriesId}>
```

Comment:
This is missleading. The `barSeries` is an array of series. Not a mapping of id to series object
```suggestion
      {barSeries.map((series) => (
        <div key={series.id}>
```